WidgetKit/TopSites/TopSitesProvider.swift

```diff
@@ -20,49 +14,55 @@ struct TopSitesProvider: TimelineProvider {
     }
 
     func getSnapshot(in context: Context, completion: @escaping (TopSitesEntry) -> Void) {
```

Comment:
Would be nice to quick clean it but no issues if we nest it with if lets

Reply:
Agreed, definitely a little messy :(